base_tier_validation/models/tier_validation.py

```diff
@@ -101,6 +101,7 @@ def _get_sequences_to_approve(self, user):
                 sequences.append(my_sequence)
         return sequences
 
+    @api.depends_context("uid")
```

Comment:
can you elaborate, why is it a missing decorator?

Reply:
forgot to ping you @grindtildeath

Reply:
@LoisRForgeFlow It's such a long time ago I don't remember exactly why it was needed. :disappointed: 

But if `self.env.user` is used in the compute function, it means the function still depends on connected user. I guess it could havve been we had some local tests where the value was not recomputed after altering the user, but I cannot tell for sure.

Reply:
it is hard for me to approve then. Without steps to reproduce or a description of what's being solved... :confused: